### PR TITLE
Change metric names from nanos to millis.

### PR DESCRIPTION
**Pull request by cancecen**

### CODE REVIEW

**Comment by cancecen** [comment]
👀 @afalko

File: src/main/java/com/salesforce/WriteTopic.java [hunk 1]
```diff
@@ -100,7 +99,7 @@ public Exception call() {
 
             // Produce one message to "warm" kafka up
             gaugeMetric(AWAITING_PRODUCE_METRIC_NAME, 1);
```
Comment (afalko) [inline src/main/java/com/salesforce/WriteTopic.java]: Do we need to change the namespaces to indicate millis too?

Comment (afalko) [reply src/main/java/com/salesforce/WriteTopic.java]: Maybe not this one specifically b/c it is counter

Comment (cancecen) [reply src/main/java/com/salesforce/WriteTopic.java]: Yeah, you probably already noticed - I changed them in BenchmarkApp.java

**Pull request merged**